LEGO1/omni/include/mxtimer.h

```diff
@@ -20,14 +20,20 @@ class MxTimer : public MxCore {
 			return g_lastTimeTimerStarted;
 		}
 		else {
-			return g_lastTimeCalculated - this->m_startTime;
+			return GetTimeSinceStart();
 		}
 	}
 
 	// SYNTHETIC: LEGO1 0x100ae0d0
 	// MxTimer::`scalar deleting destructor'
 
 private:
+	// This function appears to be public in BETA10; this function may also be
+	// an older version of GetTime() instead of a private subroutine.
+	// None of this matters for the release build since these functions are inlined.
```

Comment:
This isn't quite correct and something we have to be very careful about. The inlining patterns do matter and have an influence on the code the compiler generates. In other words, the fact that an extra inline function is now being called in `GetTime` may cause regressions in some (not necessarily all) instances where `GetTime` is called - and that's a lot of potential places since it's called in numerous functions.

If I revert `GetTime` to the way it was before, I actually receive a ~92% match on `LegoRaceCar::VTable0x94`. So I think it's more likely this function doesn't exist in retail anymore.

Reply:
Good catch! I used to have another error elsewhere that cancelled out the mismatch, so I didn't expect the incorrect stack layout to come from here. I added the BETA10 annotation. Furthermore, I added a comment that this function was likely changed between BETA10 and release.